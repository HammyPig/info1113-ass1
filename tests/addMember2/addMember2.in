MEMBER 100000
ADD MEMBER poopoo
MEMBER 100000
MEMBER 100001
add member poo poo pee pee
MEMBER 100001
aDD mEMBer i am tired
MEMBER 100002