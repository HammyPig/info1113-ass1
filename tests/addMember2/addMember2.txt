adds members (including multiple words)