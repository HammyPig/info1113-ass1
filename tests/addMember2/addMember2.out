user: No members in system.

user: Success.

user: 100000: poopoo

user: No such member in system.

user: Success.

user: 100001: poo poo pee pee

user: Success.

user: 100002: i am tired

user: Ending Library process.
